webclient/index.html

```diff
@@ -26,7 +27,7 @@
   <div id="tab-login">
     <h3>Login to server</h3>  
     <label for="host">Host</label>
-    <input type="text" id="host" value="127.0.0.1" />
+    <input type="text" id="host" value="cockatrice.woogerworks.com" />
```

Comment:
I didnt look, but if your going to expand the pre-defined host string you may want to expand the box character length as well. I know when I worked on the page at the woogerworks site I expanded the box length to fit the entire string without causing you to have to arrow over to see the right most characters.